testing/loki_alerts/Couchbase-Server/CB90070-permissionDenied/memcached.log.000000.txt

```diff
@@ -0,0 +1,2 @@
+20220120-092212/couchbase.log:cat: Permission denied
+20220120-092212/couchbase.log:cat: Permission denied
```

Comment:
This doesn't look like memcached.log to me, this looks more like couchbase.log :notsureif:

Reply:
Ah, my mistake. I was going based on the spreadsheet saying they appeared in memcached.log. I just checked and they do only appear in couchbase.log. Do you know if there's any other way to get permission denied errors because we can't access couchbase.log can we?

Reply:
They do appear in memcached.log - try firing up a Vagrant, loading a bucket, running `chown -R root:root /opt/couchbase/var/lib/couchbase/data/travel-sample`, then doing any KV ops (editing a document in the web UI should be enough)

Reply:
Thank you I'll try that now

Reply:
Fixed! Thank you :)